temporalcli/commands_test.go

```diff
@@ -216,12 +218,23 @@ func (s *SharedServerSuite) TearDownTest() {
 		s.CommandHarness.Close()
 	}
 	s.CommandHarness = nil
+	s.testVars = nil
 }
 
 func (s *SharedServerSuite) T() *testing.T                 { return s.Suite.T() }
 func (s *SharedServerSuite) SetT(t *testing.T)             { s.Suite.SetT(t) }
 func (s *SharedServerSuite) SetS(suite suite.TestingSuite) { s.Suite.SetS(suite) }
 
+func (s *SharedServerSuite) testVar(key string) string {
```

Comment:
I don't understand this suite-level helper, can't a test make whatever variables it needs?

Reply:
I don't _hate_ it, it can stay, though I don't see ever using it myself (I think passing typesafe state around via explicitly named things to be much clearer and a more natural programming thing to do)

Reply:
It is type-safe, it just happens to be all strings (we use strings as identifiers everywhere so it's useful even limited to strings). And I think it's fine for tests to use a more implicit style than I would want in non-test code.

But in this case you're right it's easy to do it the straightfoward way for these tests. That pattern has more value with more complex tests.